pkg/detectors/digitaloceanv2/digitaloceanv2.go

```diff
@@ -54,15 +56,39 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 			}
 			req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", resMatch))
 			res, err := client.Do(req)
+
 			if err == nil {
-				defer res.Body.Close()
-				if res.StatusCode >= 200 && res.StatusCode < 300 {
-					s1.Verified = true
-				} else {
-					// This function will check false positives for common test words, but also it will make sure the key appears 'random' enough to be a real key.
-					if detectors.IsKnownFalsePositive(resMatch, detectors.DefaultFalsePositives, true) {
+				switch {
+				case strings.HasPrefix(resMatch, "dor_v1_"):
+					bodyBytes, err := io.ReadAll(res.Body)
```

Comment:
This requires a different request: https://docs.digitalocean.com/reference/api/oauth-api/#refresh-token-flow